home-application/src/main/java/com/softserveinc/ita/homeproject/application/constants/Permissions.java

```diff
@@ -80,4 +80,12 @@ private Permissions() {
 
     public static final String DELETE_APARTMENT_PERMISSION = "hasAuthority('DELETE_APARTMENT_PERMISSION')";
 
+    public static final String GET_OWNERSHIP_PERMISSION = "hasAuthority('GET_OWNERSHIP_PERMISSION')";
+
+    public static final String GET_OWNERSHIPS_PERMISSION = "hasAuthority('GET_OWNERSHIPS_PERMISSION')";
+
```

Comment:
Do we need two permisions for get one and get all operations?

Reply:
moved to only one permission